Unit tests for Toshiba A/C based on captures from a remote.

crankyoldgit opened the pull request:
Data provided by @mwildbolz

crankyoldgit committed:
Unit tests for Toshiba based on captures from a remote.
Data provided by @mwildbolz

crankyoldgit commented:
@mwildbolz Any chance you could:
a) Provide a capture with the remote powering **off** the A/C unit. Pref in a non-heat mode.
b) Details of the A/C and remote units in question. e.g. model numbers.

Thanks in advance, and thanks for the data (from #340)

crankyoldgit commented:
#339

mwildbolz commented:
Hi there!
Attached you can find the captures powering on and off - [capture.txt](https://github.com/markszabo/IRremoteESP8266/files/1485878/capture.txt) - captured with IRrecvDumpV2

The type of the remote is WH-TA04NE / 70302C (this is the serial, I think).
The type of the installed Toshiba split device is: RAS-B13N3KVP-E
I have different Toshiba devices installed, I'll check with all of them...

BR,
Markus

crankyoldgit committed:
Update unit test for a 'real' power off example.

crankyoldgit committed:
Update working status and known-working models etc.

crankyoldgit commented:
Thanks @mwildbolz. I've added the 'OFF' case to the unit tests. That really is odd behaviour IMHO, but hey, it is what it is. :-/
I've also added the model info to. Thanks for all your help and fast turnaround on requests.

mwildbolz commented:
Hi David!


I also got some other Toshiba A/C remote (from work) and tested it with the library - it is working!

Tested remote type: WC-L03SE

Corresponding A/C unit: Toshiba RAS 18SKP-ES



BR,

Markus


________________________________
Von: David Conran <notifications@github.com>
Gesendet: Freitag, 17. November 2017 10:21
An: markszabo/IRremoteESP8266
Cc: Wildbolz, Markus; Mention
Betreff: Re: [markszabo/IRremoteESP8266] Unit tests for Toshiba A/C based on captures from a remote. (#346)


@mwildbolz<https://github.com/mwildbolz> Any chance you could:
a) Provide capture with the remote powering off the A/C unit. Pref in the non-heat mode.
b) Details of the A/C and remote units in question. e.g. model numbers.

Thanks in advance, and thanks for the data (from #340<https://github.com/markszabo/IRremoteESP8266/pull/340>)

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/markszabo/IRremoteESP8266/pull/346#issuecomment-345189141>, or mute the thread<https://github.com/notifications/unsubscribe-auth/Af04J1-WWyHAiNYUu9Bds4HZ4nahmkUSks5s3VAPgaJpZM4Qhv8Q>.

crankyoldgit committed:
Update Toshiba compatiablity list.
@mwildbolz reported it works these particular models.

crankyoldgit commented:
Thanks @mwildbolz. This PR has been updated with that info.

crankyoldgit commented:
It's been a week since the initial review request. Friendly reviewer ping. :-)

sillyfrog reviewed (approved):
Tests pass - winner! :)
